vignettes/faq.Rmd

```diff
@@ -26,12 +26,12 @@ knitr::opts_chunk$set(eval = FALSE)
 Please cite Keras in your publications if it helps your research. Here is an example BibTeX entry:
 
 ```
-@misc{chollet2015keras,
-  title={Keras},
-  author={Chollet, Fran\c{c}ois and others},
-  year={2015},
+@misc{allaire2017keras,
```

Comment:
I think the citation should still be chollet2015keras and that his name should be listed first (assuming it's not automatically alphabetical)